packages/api/swagger-config.json

```diff
@@ -1,171 +1,183 @@
+                  "content": {
+                     "application/json": {
+                        "schema": {
+                           "$ref": "#/components/schemas/Id"
+                        }
+                     }
+                  }
+               },
+               "400": {
+                  "$ref": "#/components/responses/badRequest"
+               },
+               "5XX": {
+                  "$ref": "#/components/responses/internalServerError"
+               }
+            }
+         },
+         "get": {
+            "tags": [
+               "W3Name HTTP API"
+            ],
+            "summary": "Retrieve a record",
+            "description": "Retrieve an IPNS Record.",
+            "operationId": "get-record",
+            "security": [],
+            "parameters": [
+               {
+                  "name": "key",
+                  "in": "path",
+                  "required": true,
+                  "schema": {
+                     "$ref": "#/components/schemas/Key"
+                  }
+               }
+            ],
+            "responses": {
+               "200": {
+                  "description": "OK",
+                  "content": {
+                     "application/json": {
+                        "schema": {
+                           "$ref": "#/components/schemas/Record"
+                        }
+                     }
+                  }
+               },
+               "400": {
+                  "$ref": "#/components/responses/badRequest"
+               },
+               "5XX": {
+                  "$ref": "#/components/responses/internalServerError"
+               }
+            }
+         }
+      }
+   },
+   "components": {
+      "schemas": {
+         "Id": {
+            "type": "object",
+            "properties": {
+               "id": {
+                  "$ref": "#/components/schemas/Key"
+               }
+            },
+            "description": "The published IPNS Key that will rersolve the record once it has persisted to th DHT."
+         },
+         "Record": {
+            "type": "object",
+            "properties": {
+               "value": {
+                  "type": "string",
+                  "example": "/ipfs/QmPqrEHJTex2CPbqNULCmbSFJT3boBwAAfMb5UjvXtKjEe"
+               },
+               "record": {
+                  "type": "string",
+                  "example": "CjQvaXBmcy9RbVBxckVISlRleDJDUGJxTlVMQ21iU0ZKVDNib0J3QUFmTWI1VWp2WHRLakVlEkB/s3F4iHVMyA9LOFizt5N4PHiI2APD15wsMa1RXGwmugThVpUjTw+5HHwFt16TI/8AKk/1p8+26hylxMxd3wAIGAAiHjIwMjMtMDgtMDVUMDk6NDk6MjguNDk5MDAwMDAwWigNMODyh77rhc0FQkCdgM1u1AAUsdf4pE7/kNYjl438/g6Ja48Tr2evFSRc6YSMgljaKX8PpQC0JDlZqF2jiKdHRUOaxLHArPo9MW0PSosBpWNUVEwbAAs0LrfB+WBlVmFsdWVYNC9pcGZzL1FtUHFyRUhKVGV4MkNQYnFOVUxDbWJTRkpUM2JvQndBQWZNYjVVanZYdEtqRWVoU2VxdWVuY2UNaFZhbGlkaXR5WB4yMDIzLTA4LTA1VDA5OjQ5OjI4LjQ5OTAwMDAwMFpsVmFsaWRpdHlUeXBlAA=="
+               }
+            },
+            "description": "an IPNS Record and value."
```

Comment:
I think that technically the record contains the value anyway (along with the expiry, seqno and signature), so saying "an IPNS Record and value" is a little bit misleading.

Reply:
Aye that's a great point! Addessed.